src/main/java/com/github/instagram4j/instagram4j/requests/usertags/UserTagsFeedRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.github.instagram4j.instagram4j.requests.usertags;
+
+
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.requests.IGGetRequest;
+import com.github.instagram4j.instagram4j.requests.IGPaginatedRequest;
+import com.github.instagram4j.instagram4j.responses.usertags.UserTagsFeedResponse;
+
+import lombok.AllArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+public class UserTagsFeedRequest extends IGGetRequest<UserTagsFeedResponse> 
+					implements IGPaginatedRequest {
+	
+    private long userId;
+    
+    @Setter private String max_id ="";

```

Comment:
Put annotation above the field instead of next to the field. Remove instantiation it is okay to leave it null.

See [FeedTagRequest](https://github.com/instagram4j/instagram4j/blob/master/src/main/java/com/github/instagram4j/instagram4j/requests/feed/FeedTagRequest.java). Style and formatting is similar.